### PR TITLE
SAKURA instead of SNOW since it is April:P ##panels

### PR DESCRIPTION
**Pull request by Vane11ope**
GIFrecord_2019-04-03_205911

Yup. I like it Magenta for this season to make it look like SAKURA because I'm from Japan.

### CODE REVIEW

File: libr/core/panels.c [hunk 1]
```diff
@@ -2261,7 +2261,11 @@ static void printSnow(RPanels *panels) {
 			continue;
 		}
 		if (r_cons_canvas_gotoxy (panels->can, snow->x, snow->y)) {
-			r_cons_canvas_write (panels->can, "*");
+			if (panels->fun == PANEL_FUN_SAKURA) {
+				r_cons_canvas_write (panels->can, Color_BMAGENTA"*"Color_RESET);
```
Comment (radare) [inline libr/core/panels.c]: leafs are not asterisks

Comment (Vane11ope) [reply libr/core/panels.c]: lmao right. but in Japan, we only focus on beautiful SAKURA flower falls and dismiss leaf anyway :P

**Comment by radare** [comment]
Maybe a comma represents better

**Comment by Vane11ope** [comment]
let me try

**Comment by Vane11ope** [comment]
holy shit you are right im taking gif just wait

**Comment by Vane11ope** [comment]
GIFrecord_2019-04-03_225629
check this out

**Comment by Vane11ope** [comment]
yeah this looks more sakura than *, let me send this code right now

**Pull request merged**